docs/en/observability/synthetics-recorder.asciidoc

```diff
@@ -0,0 +1,144 @@
+// - The reader might be using Project monitors, Heartbeat, or Elastic Agent.
+// 
+// What's not included:
+// - Detailed definitions of journey, step, etc.
+// - Detailed instructions on how to create a monitor. Point to instructional guide instead.
+
+[[synthetics-recorder]]
+= Synthetics Recorder
+
+You can use the Synthetics Recorder to <<synthetics-create-test, write a synthetic test>> by interacting with a web page and exporting journey code that reflects all the actions you took.
+
+[discrete]
+== Set up
+
+Download and install the Elastic Synthetics Recorder from the https://github.com/elastic/synthetics-recorder/releases[GitHub repository's release page].
+
+[discrete]
+== Record a journey
+
+To record a journey:
+
+. Enter a starting URL in the search box. This URL will be the starting point of the journey script the recorder will create.
+. Click **Start** or press Enter on your keyboard. This will launch a Chromium window open to the page you specified and start recording.
+. Start interacting with the browser. This can include clicking on text, navigation, focusing on inputs like buttons and text fields, and more.
+  . (Optional) You can click **Pause** to temporarily stop recording actions while you continue to interact with the browser. Click again to start recording actions again. Note: It's especially important to <<synthetics-recorder-test-the-journey, test the journey>> if you paused recording at any point.
+. When you're done interacting with the browser window, click **Stop** to stop recording.
+// . As you complete your journey, you will see the actions you are generating populate in the Script Recorder's window.
+
+[discrete]
+[[synthetics-recorder-edit-a-journey]]
+== Edit a journey
+
+Once you've started recording, you can use the Synthetics Recorder UI to edit steps and individual actions before generating the journey code.
+You can also edit the journey after you've stopped recording.
+
+[discrete]
+=== Name steps
+
+Naming steps can help make the resulting journey code easier to understand.
+If you provide a step name, the name will be used in both the UI and the resulting code.
+If you don't name steps, the UI will show "Step 1", "Step 2", and so on, and the resulting code will use the first action in the step as the step text.
+
+To edit a step name:
+
+. Hover over the current step name and click the pencil icon that appears.
+. Edit the text in the text box.
+. Click Return or Enter on your keyboard to save the updated name.
+
+[discrete]
+=== Split into multiple steps
+
+Steps represent groups of actions that should be completed in a specific order.
+Breaking a journey into steps can make it easier to read the resulting code.
+It can also make it easier to interpret results in the {uptime-app} since each step is
+displayed individually in the {uptime-app} along with screenshots for convenient debugging and error tracking.
+
+By default, the Synthetics Recorder will group all actions in a single step,
```

Comment:
Code generation logic sometimes decides to divide some steps! :D (for example when navigating to a new website etc). correct me if I'm wrong @justinkambic

Reply:
@kyungeunni you're correct - the code generator will not split steps anymore as we have updated the app. Now when you record, all actions are "bucketed" into the first step, and users can split them up into whichever step configurations they desire. So the original text here is correct.